docs/setup/piweb_auto_importer.md

```diff
@@ -6,31 +6,38 @@ title: PiWeb Auto Importer
+## Development mode
+*PiWeb Auto Importer* supports a special development mode that enables additional features useful for developing plug-ins. For security reasons, these features are not enabled by default in a standard installation of *PiWeb Auto Importer*. To activate the development mode, you need to create a specific value in the windows registry:
 
 In `HKEY_LOCAL_MACHINE\SOFTWARE\Zeiss\PiWeb\AutoImporter`, if it does not yet exist, create a new *string* value `DevelopmentMode` and set it to `true`.
 
 ![Editing development mode in Windows registry editor](../../assets/images/setup/piweb_auto_importer/regedit_development_mode.png "Editing development mode in Windows registry editor"){: .framed }
 
-*PiWeb Auto Importer* will now show [Development mode] in its title bar.
+After the next start, *PiWeb Auto Importer* will now show [Development mode] in its title bar to indicate that the development mode is now active.
 
 ![Active development mode in PiWeb Auto Importer](../../assets/images/setup/piweb_auto_importer/titlebar_development_mode.png "Active development mode in Auto Importer")
 
-With active development mode, it is possible to specify additional plug-in search paths via command line parameter <span class="nowrap">`-pluginSearchPaths`</span> when starting *PiWeb Auto Importer*. The typical usage of this parameter is to set the plug-in search path directly to the output folder of your plug-in project.
+## Plug-in search paths
+Import plug-ins are usually installed via *.pip* files to a specific plug-in installation folder. However, since the plug-in installation is system wide, administrator privileges would be required for each change. At the customer side this is desired for security reasons, however, this is not practical for plug-in development at all. For this reason, *PiWeb Auto Importer* allows to load plug-ins from custom unprivileged locations by specifying additional plug-in search paths via command line parameter <span class="nowrap">`-pluginSearchPaths`</span>. The typical usage of this parameter is to set the plug-in search path directly to the output folder of your plug-in project. Plug-in projects usually have a `launchSettings.json` configured to do this when the project is launched.
 
 ![PiWeb Auto Importer command line example](../../assets/images/setup/piweb_auto_importer/cmd_plugin_search_path.png "PiWeb Auto Importer command line example")
 
+{: .note }
+The <span class="nowrap">`-pluginSearchPaths`</span> command line parameter is only available when *PiWeb Auto Importer* is in development mode. See above for how to active development mode.
+
 ## Plug-in management
 To see whether plug-ins in custom search paths are found, you can compile one of the example plugins, use its build output path as custom search path and open the plugin management view.

```

Comment:
I seem to have used plugin instead of plug-in here. I would be thankful if you could adjust it.

Reply:
Will do.